src/utils/rttc.ts

```diff
@@ -37,6 +37,9 @@ const typeOf = (v: Value) => {
 const isNumber = (v: Value) => typeOf(v) === 'number'
```

Comment:
The above,

https://github.com/source-academy/js-slang/blob/b5d72aed774cc2d58ed7d474c3ad371b73f066e0/src/utils/rttc.ts#L30-L36

can be removed to simply use the typeof operator, as Closures are now a subclass of Function.

Reply:
Rewrote typeOf to show 'null' and 'array' for `null` and arrays respectively, removed special case for function.